microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -60,10 +60,21 @@ The Microsoft Edge team will independently decide on MV3 migration timelines for
 | TBD | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge, by using enterprise policies. |
 | TBD | No change. | Manifest V2 extensions will no longer function in Microsoft Edge, even with the use of enterprise policies. |
 
-Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.
+
+<!-- ------------------------------ -->
+#### Timeline for MV2-based extensions
+
+The move to Manifest V3 improves the security, privacy, and performance of extensions.  Our current strategy for Microsoft Edge is to support new MV3-based extensions and support updates to existing MV2-based extensions from our partner ecosystem.  Microsoft Edge Extensions will continue to support updates for MV2-based extensions for an extended period of time, and we will communicate with our partner ecosystem when we plan the deprecation of existing MV2-based extensions.  For enterprises, Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.
```

Comment:
the initial draft text: 
MV2 timeline for Enterprise: 
We see the move to MV3 as a change in the positive direction to improve security, privacy and performance of extensions. Our current strategy is to honour any new mv3 extensions from our partner ecosystem and any updates to existing mv2 extensions. At the same time, we will continue updates for MV2 extensions for an extended period of time and will communicate once we plan deprecation of existing mv2 extensions. For enterprises, Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.

@sivMSFT